service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -41,6 +44,10 @@
     private ExecutorService executor;
     private IotHubConnectionString iotHubConnectionString;
 
+    @Setter
```

Comment:
Is there a standard order in which to list these? I'd have assumed get would be first, if there was.

Reply:
There isn't a standard that I know of, unlike in C#. I can switch them if it's bugging you though